kvmagent/tests/test_cpu_hotplug_online.py

```
@@ -1,143 +1,138 @@
+        self.assertEqual(len(offline_cpus), 0)
 
-    def test_qga_exception_is_swallowed(self):
-        """QGA failure must not affect hotplug - exception should be caught."""
+    def test_exception_swallowed(self):
+        """QGA exceptions must not propagate."""
         caught = False
         try:
-            # Simulate the try/except pattern from _qga_online_hotplugged_cpus
-            raise Exception("QGA connection timeout")
+            raise Exception("QGA timeout")
         except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)